src/UI/Components/Diagram/components/FormModal.tsx

```diff
@@ -288,7 +292,7 @@ const FormModal = ({
               variant="info"
               isInline
               title={
-                cellView
+                cellToEdit
                   ? words("inventory.editInstance.noAttributes")
```

Comment:
maybe adjust the naming here for the text.  it's a bit confusing having addInstance/editInstance mixed.

Reply:
like above, I'll link the comments though in the ticket to the right sidebar